### PR TITLE
VIH-7438 message icons does not show when participant has IM'ed

### PR DESCRIPTION
**Pull request by williambowshersolirius**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/VIH-7438

### Change description ###
Modified the SendMessage function so that the participant id is delivered to the UI, so that when the unread messages are checked the Id can be used to lookup any unread messages and display the msg icon in the admin dashboard.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/unread-messages/unread-messages.component.ts [hunk 1]
```diff
@@ -29,6 +29,7 @@ export class UnreadMessagesComponent extends UnreadMessagesComponentBase impleme
     }
 
     ngOnInit() {
+        console.log();
```
Comment (levibutler) [inline VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/unread-messages/unread-messages.component.ts:32]: removed?

File: VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/command-centre/command-centre.component.ts [hunk 2]
```diff
@@ -209,6 +209,7 @@ export class CommandCentreComponent implements OnInit, OnDestroy {
         this.loadingData = reload;
         this.conferencesSubscription = this.queryService.getConferencesForVHOfficer(this.venueAllocations).subscribe(
             async (data: ConferenceForVhOfficerResponse[]) => {
+                console.log('calling get conferences');
```
Comment (levibutler) [inline VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/command-centre/command-centre.component.ts:212]: remove or log using the logger

File: VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/unread-messages/unread-messages.component.ts (hunk 1 above)
Comment (williambowshersolirius) [reply VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/unread-messages/unread-messages.component.ts]: removed

File: VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/command-centre/command-centre.component.ts (hunk 2 above)
Comment (williambowshersolirius) [reply VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/command-centre/command-centre.component.ts]: removed

**Review by RuwanikaWeerakkody — APPROVED** [review]

**Pull request merged**